site/diagrams/sequenceDiagrams/command_control_Success.txt

```diff
@@ -2,9 +2,9 @@ participant AMQP 1.0 Network
 participant Business Application
 
 Business Application->AMQP 1.0 Network: attach("sender","control/${tenant_id}/${device_id}")
-Business Application->AMQP 1.0 Network: attach("receiver","control/${tenant_id}/${device_id}/${reply-to}")
+Business Application->AMQP 1.0 Network: attach("receiver","control/${tenant_id}/${reply-to}")
```

Comment:
what about the PNG generated from this?

Reply:
Good question. I changed it, but now it is gone. Thanks for spotting!